<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>metadata_details</class>
 <widget class="QWidget" name="metadata_details">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>675</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>565</width>
        <height>825</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QgsCollapsibleGroupBox" name="gbp_description">
         <property name="title">
          <string>Description</string>
         </property>
         <layout class="QGridLayout" name="gl_description">
          <item row="0" column="2">
           <widget class="QLineEdit" name="le_title">
            <property name="toolTip">
             <string>Characteristic and often unique name by
        which the resource is known</string>
            </property>
            <property name="inputMask">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lbl_context">
            <property name="toolTip">
             <string>Datastore Id</string>
            </property>
            <property name="text">
             <string>Context</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lbl_thematic">
            <property name="text">
             <string>Thematics</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="lbl_inspire_kw">
            <property name="text">
             <string>INSPIRE Keywords</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbl_description">
            <property name="toolTip">
             <string>Brief narrative summary of the resource
        content</string>
            </property>
            <property name="text">
             <string>Description</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lbl_unique_id">
            <property name="toolTip">
             <string>Value uniquely identifying the resource
        in the catalog</string>
            </property>
            <property name="text">
             <string>Unique Id</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_title">
            <property name="toolTip">
             <string>Characteristic and often unique name by
        which the resource is known</string>
            </property>
            <property name="text">
             <string>Title</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="lbl_kw">
            <property name="text">
             <string>Keywords</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="le_context">
            <property name="toolTip">
             <string>Datastore Id</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="le_unique_id">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Value uniquely identifying the resource in the catalog. Must start with '_' or letter and can contains letters, numbers, '-', '_' or '.'&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QLineEdit" name="le_kw"/>
          </item>
          <item row="1" column="2">
           <widget class="TextEditPreviewWidget" name="te_description" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBox" name="gbp_quality">
         <property name="title">
          <string>Quality</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <widget class="QLineEdit" name="le_genealogy"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbl_genealogy">
            <property name="text">
             <string>Genealogy</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBox" name="gbp_temporal">
         <property name="title">
          <string>Temporal References</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QDateEdit" name="de_creation_date">
            <property name="dateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>1800</year>
              <month>1</month>
              <day>1</day>
             </datetime>
            </property>
            <property name="minimumDateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>1800</year>
              <month>1</month>
              <day>1</day>
             </datetime>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_creation_date">
            <property name="text">
             <string>Creation date</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbl_frequency">
            <property name="text">
             <string>Frequency</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cb_frequency">
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::NoInsert</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBox" name="gbp_contact">
         <property name="title">
          <string>Contacts</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_contact_email">
            <property name="text">
             <string>Email</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="le_contact_email"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBox" name="gbp_property">
         <property name="title">
          <string>Resource manager</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_org_name">
            <property name="text">
             <string>Organization</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbl_org_email">
            <property name="text">
             <string>Email</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="le_org_name"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="le_org_email"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBox" name="gbp_info">
         <property name="title">
          <string>Metadata Informations</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="5" column="0">
           <widget class="QLabel" name="lbl_link">
            <property name="text">
             <string>Metadata link</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="cb_encoding">
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::NoInsert</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_type">
            <property name="text">
             <string>Resource type</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lbl_language">
            <property name="text">
             <string>Metadata language</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="lbl_encoding">
            <property name="text">
             <string>Encoding</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="le_link"/>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="cb_language">
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::NoInsert</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cb_type"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TextEditPreviewWidget</class>
   <extends>QWidget</extends>
   <header>geoplateforme.gui.wdg_text_edit</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
